spring/boot-autoconfigure/src/main/java/com/linecorp/armeria/spring/DocServiceConfigurator.java

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright 2019 LINE Corporation
```

Comment:
Could you make the copyright header look same with other files? i.e.

```java
/*
 * Copyright ...
 *
 * ...
 */
```

Reply:
Done!